config/galaxy.ini.sample

```diff
@@ -1296,11 +1296,13 @@ use_interactive = True
 # a reasonable size.
 #max_metadata_value_size = 5242880
 
-# If (for example) you run on a cluster and your datasets (by default,
-# database/files/) are mounted read-only, this option will override tool output
-# paths to write outputs to the working directory instead, and the job manager
-# will move the outputs to their proper place in the dataset directory on the
-# Galaxy server after the job completes.
+# This option will override tool output paths to write outputs to the working
+# directory (instead of to the file_path) and the job manager will move the
+# outputs to their proper place in the dataset directory on the Galaxy server
+# after the job completes. This is necessary (for example) you run on a cluster
+# and your datasets (file_path) can not be written by the user running the jobs
```

Comment:
s/and your datasets (file_path) can not be written/and datasets are not writable/

Reply:
I'd say: "and the output datasets cannot be created inside file_path by the user running the jobs"